server/src/routes/items.routes.ts

```diff
@@ -122,4 +138,22 @@ router.post('/', authenticate, validateAddItemRequest(), async (req: Request, re
     res.status(statusCode).send(response);
 });
 
+router.put('/edit/:itemId/', authenticate, async (req: Request, res: Response) => {
```

Comment:
Add validations for:
- itemId exists in DB with the given userId. Reuse it from DELETE method.
- validation of item fields: name, imageUrl, categoryId. Reuse it from POST method (maybe rename the existing validation to something else).